project-base/src/Controller/Front/CartController.php

```diff
@@ -275,14 +284,16 @@ public function addProductAjaxAction(Request $request)
 
                 $this->sendAddProductResultFlashMessage($addProductResult);
 
-                $accessories = $this->listedProductViewFacade->getAccessories(
-                    $addProductResult->getCartItem()->getProduct()->getId(),
-                    self::AFTER_ADD_WINDOW_ACCESSORIES_LIMIT
-                );
+                $accessories = [];
+                if ($this->moduleFacade->isEnabled(ModuleList::ACCESSORIES_ON_BUY)) {
+                    $accessories = $this->listedProductViewFacade->getAccessories(
+                        $addProductResult->getCartItem()->getProduct()->getId(),
+                        self::AFTER_ADD_WINDOW_ACCESSORIES_LIMIT
+                    );
+                }
```

Comment:
It would be really lovely if accessories is provided via separated method. Do not you think?

Reply:
Well I do not see this as important. This PR is bugfix, no need to do refactoring here